src/libsync/discoveryphase.h

```diff
@@ -283,6 +283,8 @@ class DiscoveryPhase : public QObject
     // output
     QByteArray _dataFingerprint;
     bool _anotherSyncNeeded = false;
+    QHash<QString, long long> _filesNeedingScheduledSync;
```

Comment:
@claucambra Wouldn't `qlonglong` work?

Reply:
Since the method returns a `qint64` I will use that, though they are all just aliases of `long long`

Reply:
Sure :)